dev-python/e4u/e4u-0.1_rc4.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+MY_PV="${PV/_/}"
+
+DESCRIPTION="A library for handling unicode emoji and carrier's emoji"
+HOMEPAGE="https://github.com/lambdalisue/e4u"
+SRC_URI="https://github.com/lambdalisue/e4u/archive/${MY_PV}.tar.gz -> ${PN}-${MY_PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/beautifulsoup:python-2[${PYTHON_USEDEP}]
+	dev-python/setuptools"
```

Comment:
PYTHON_USEDEP!111111oneoneoneelven

Reply:
I could swear, I had done this :P Sorry, fixed now. I need that vacation now x)